LoopKit/GlucoseKit/GlucoseMath.swift

```diff
@@ -10,7 +10,7 @@ import Foundation
 import HealthKit
 
 public struct GlucoseMath {
-    public static let momentumDataInterval: TimeInterval = .minutes(15)
+    public static let momentumDataInterval: TimeInterval = .minutes(16)
```

Comment:
Was this update intentional?

Reply:
Yes, but this probably deserves some more highlighting. This is fixing an issue where momentum is dropped due to cgm sensor timestamp jitter. I'll note this in a new ticket.

Reply:
Actually, going to revert this. This does change some behavior unintentionally.